lnrpc/rpc.proto

```diff
@@ -743,90 +743,97 @@ message HTLC {
     uint32 expiration_height = 4 [json_name = "expiration_height"];
 }
 
-message ActiveChannel {
+message Channel {
     /// Whether this channel is active or not
     bool active = 1 [json_name = "active"];
 
+    /// Whether this channel is advertised to the network or not
+    bool public = 2 [json_name = "public"];
```

Comment:
in `OpenChannelRequest` we are using the `private` flag. Maybe use that here as well, to be consistent? :)